tests/integration_tests.rs

```diff
@@ -10,6 +10,7 @@ use std::ops::{Range, RangeFull};
 type StrSpan<'a> = LocatedSpan<&'a str>;
 type BytesSpan<'a> = LocatedSpan<&'a [u8]>;
 
+#[cfg(feature = "alloc")]
```

Comment:
Should be:

```
 #[cfg(any(feature = "std", feature = "alloc"))]
```

because std does not imply alloc.

(same for the other occurences below)

Reply:
Done